### PR TITLE
(Bot route) status feature + discord API information 

### PR DESCRIPTION
**Pull request by CreeperPlanet26**

### CODE REVIEW

File: server/api/bots.js [hunk 1]
```diff
@@ -18,12 +19,13 @@ router.get("/:id", async (req, res) => {
 });
 //posts a bot to the db
 router.post("/", async (req, res) => {
-    const { id, name, prefix, description, owners, website, helpCommand, supportServer, library } = req.body;
```
Comment (Maxisthemoose) [inline server/api/bots.js:22]: Why'd you remove name?

**Comment by CreeperPlanet26** [comment]
I think this is done. Please review.

File: server/api/bots.js [hunk 2]
```diff
@@ -2,29 +2,30 @@ const express = require("express");
...
-    if (!id || !name || !prefix || !description || !owners || !website || !helpCommand || !supportServer || !library) return res.status(404).json({ msg: "Your missing some information to create the bot!" });
+    const { id, prefix, description, owners, website, helpCommand, supportServer, library } = req.body;
+    if (!id || !prefix || !description || !owners || !website || !helpCommand || !supportServer || !library) return res.status(404).json({ msg: "Your missing some information to create the bot!" });
     const bot = await Bots.findOne({ id });
-    //need a good error code
+    const botApiData = await getBotData(id);
+    const { tag, avatarUrl } = botApiData;
     if (bot) return res.status(400).send({ message: "This bot already exists!", error: "Bad Request." });
-    const newBot = new Bots({ id, name, prefix, description, owners, website, helpCommand, supportServer, library });
+    const newBot = new Bots({ id, tag, avatarUrl, prefix, description, owners, website, helpCommand, supportServer, library });
```
Comment (CreeperPlanet26) [inline server/api/bots.js:28]: cuz im getting the full tag from here. no need for name anymore

File: server/api/bots.js (hunk 1 above)
Comment (CreeperPlanet26) [reply server/api/bots.js:22]:
```js
 const botApiData = await getBotData(id);
    const { tag, avatarUrl } = botApiData;
    if (bot) return res.status(400).send({ message: "This bot already exists!", error: "Bad Request." });
    const newBot = new Bots({ id, name, prefix, description, owners, website, helpCommand, supportServer, library });
    const newBot = new Bots({ id, tag, avatarUrl, prefix, description, owners, website, helpCommand, supportServer, library });

```
I am getting the full tag from here using the discord api so we dont need a name

File: server/api/bots.js (hunk 2 above)
Comment (Maxisthemoose) [reply server/api/bots.js:28]: Ok

**Pull request merged**